src/Integration.Vsix/source.extension.vsixmanifest

```diff
@@ -13,7 +13,7 @@
     <Tags>SonarLint, SonarQube, Analysis, Roslyn, CodeAnalysis, Analyzer, Code analysis, Sonar, Debt, Technical, Tech, Quality</Tags>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0]" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0.25420.00,15.0)" />
```

Comment:
I have made the tests and this change is properly working (i.e. cannot install on earlier versions).

Reply:
Why limit to 15.0? In this case we will have to release a special version supporting 16.0

Reply:
I prefer to allow installation on all versions of VS2015 and then display the message "install VS update 3 or this won't work. If you don't want to update, use this old version of SLVS".

This gives users a chance to update or consciously use older version. If we don't offer it, it may not be clear to them why the extension doesn't offer updates.

Reply:
We already have a different VSIX for VS2017 so I will ignore this comment. Besides, I would prefer to do a manual operation of enabling support for a different VS (at least we can test) rather than saying it works for everything when we don't know.

Reply:
@Evangelink What about 2015 <U3?

Reply:
Doesn't understand the question